streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamTransform.java

```diff
@@ -24,24 +24,24 @@
 import org.apache.kafka.streams.processor.ProcessorContext;
 import org.apache.kafka.streams.processor.ProcessorSupplier;
 
-public class KStreamTransform<K, V, K1, V1> implements ProcessorSupplier<K, V> {
+public class KStreamTransform<Ki, Vi, Ko, Vo> implements ProcessorSupplier<Ki, Vi> {
```

Comment:
See my other comment about the actual name choice here. But also, I'm not sure if the increase of clarity here is worth the extra review time that including it in the diff incurs.

Reply:
I think the increase in clarity is worth the (small) extra review time, especially for newbies like me that are not used to the common patterns in the code base you described in another comment.